src/i18n/translate/en/login.json

```diff
@@ -0,0 +1,15 @@
+{
+  "emailRequired": "Email is a required field",
+  "emailValid": "Email must be a valid email",
+  "emailMaxCharacter": "Email must be at most 50 characters",
```

Comment:
đặt translate gần sát với ngữ nghĩa

Email must be at most 50 characters -> email_must_50_char / email_length_50